ojd_daps_skills/getters/download_public_data.py

```diff
@@ -14,5 +13,43 @@ def download():
     os.system(f"rm {public_data_dir}.zip")
 
 
+if __name__ == "__main__":
+    download()
+
+from ojd_daps_skills import PUBLIC_DATA_FOLDER_NAME, PROJECT_DIR
+
+import os
+import boto3
+from botocore.exceptions import ClientError
+from botocore import UNSIGNED
+from botocore.config import Config
+from zipfile import ZipFile
+
+
+def download():
```

Comment:
i'm unsure if you've realised but you've defined and called both the aws cli command aswell as using boto to download the data?
You've also overwritten the same function

Reply:
Brilliant spot! I had a long day. Thank you 🤣